tests/resumable_media/unit/requests/test__helpers.py

```diff
@@ -63,345 +57,3 @@ def test__get_body_w_content_consumed(self):
 
 def _make_response(status_code):
     return mock.Mock(status_code=status_code, spec=["status_code"])
-
-
-def _get_status_code(response):
-    return response.status_code
-
-
-class Test_wait_and_retry(object):
```

Comment:
Amazing ✨  Do we need to replace / add any retry tests? Or are the retry conformance tests sufficient?

Reply:
We have 100% coverage. Conformance tests pass. We can't reliably test retries in a system test scenario. So I think it's as tested as we can make it.